test/java/seedu/addressbook/data/person/NameTest.java

```diff
@@ -9,10 +9,22 @@
 public class NameTest {
 
     @Test
+    public void testConstructor_throwIllegalValueExceptionIfContainNonAlphabeticCharacters() {
+        try {
+            Name name = new Name("1234!!");
+            fail("Should throw IllegalValueException");
+        } catch (IllegalValueException ive) {
```

Comment:
in general you can make tests as `void test_x_y_z throws <someException> {` no need to `try ... catch` in the test